soda-fountain-lib/src/main/scala/com/socrata/soda/server/persistence/pg/PostgresStoreImpl.scala

```diff
@@ -27,6 +27,153 @@ import org.joda.time.DateTime
 
 case class SodaFountainStoreError(message: String) extends SodaInternalException(message)
 
+object PostgresStoreImpl {
+  def fetchMinimalColumnsSql (includeColumnFilter: Boolean, isDeleted: Boolean) = {
+    val columnFilter = if (includeColumnFilter) "c.column_id = ? AND" else ""
+    val deletedFilter = if (!isDeleted) "AND dc.deleted_at is null" else "AND dc.deleted_at is not null"
+    s"""SELECT c.column_name,
+       |       c.column_id,
+       |       c.type_name,
+       |       c.is_inconsistency_resolution_generated,
+       |       cs.computation_strategy_type,
+       |       cs.source_columns,
+       |       cs.parameters
+       | FROM columns c
+       | JOIN dataset_copies dc on dc.id = c.copy_id
+       | LEFT JOIN computation_strategies cs
+       | ON c.dataset_system_id = cs.dataset_system_id AND c.column_id = cs.column_id AND c.copy_id = cs.copy_id
+       | WHERE $columnFilter
+        |dc.dataset_system_id = ? AND
+        |       dc.copy_number = ?
+        |       $deletedFilter
+        |       """.stripMargin
```

Comment:
```suggestion
       | dc.dataset_system_id = ? AND
       |        dc.copy_number = ?
       |        $deletedFilter
       |        """.stripMargin
```
May as well fix it now.